api/lanes.py

```diff
@@ -235,6 +228,13 @@ def create_lanes_for_large_collection(_db, library, languages, priority=0):
     YA = [Classifier.AUDIENCE_YOUNG_ADULT]
     CHILDREN = [Classifier.AUDIENCE_CHILDREN]
 
+    common_args = dict(
+        languages=languages,
+        media=[Edition.BOOK_MEDIUM]
```

Comment:
might be good to change this to `Edition.FULFILLABLE_MEDIA` now that it exists

Reply:
I thought about that, and it might end up that way, but it's more likely we'll end up putting audiobooks in separate lanes, which would have a different `media` setting.

Reply:
That's true, it would be better to have audiobooks in different lanes.

Reply:
We're doing a survey to see how people want to encounter different types of books. Certainly movies would be in a separate section.